### ISSUE TITLE
JSON file not saving the content.

### ISSUE THREAD

**Issue by seigel** [zadam/trilium#307]
https://d.pr/i/F3Lhhj

Navigate away...

https://d.pr/i/YSoUrt

Come back and gone

https://d.pr/i/B52AcV

VERISON INFO

https://d.pr/i/Tjk8Y7

**Comment by zadam** [comment]
This is very strange. Everything looks correct - no errors in the dev console, there's a check in the upper right corner showing that changes have been saved ... Does it happen everytime of is it a one time problem?

**Comment by seigel** [comment]
Seems to be repeatable...I'll try restarting the app and see what happens. Restarted and same issue.  (Mac OSX)

**Comment by zadam** [comment]
I will try to do some testing on Mac as well.

**Comment by seigel** [comment]
If I create another note, it seems to save.  Something special about that one note...Did you need me to pack anything up and send it to you to see what is interesting about that note?

**Comment by zadam** [comment]
If you don't have any sensitive data in the document, you could pack it and send it to me. It's the "document.db" file in the /Users/[user]/Library/Application Support/trilium-data directory. My email is adam.zivner@gmail.com

**Comment by seigel** [comment]
Sent

**Comment by zadam** [comment]
Thank you! I was able to reproduce the problem and fix the issue thanks to your document.

This bug occurs only when you try to save invalid JSON (as was your example with "...") into an already existing JSON code note. If you attempt to save a valid JSON then it works correctly. But of course this is incorrect behavior and Trilium should allow saving invalid JSON without an issue. So this was fixed and will be in the next release.
